### ISSUE TITLE
Acrobat reports missing alt text

### ISSUE THREAD

**Issue by ntderosu-code** [ntderosu-code/PDF-Assist11y#108]
### Problem

This was supposedly fixed, but it is now. See #86 

### Possible solution

Acrobat should correctly report alt text. Properly handle any additional parent tags as needed. Ensure that changes to not create content flow / MCAS issues.

### References

#86
